### PR TITLE
Edwin

### PR DESCRIPTION
**Pull request by krayzieeddi**
It's still not finished but I just wanted you guys to see the progress so far. Also still waiting on the others to add their candidate to the google doc. Any comments or changes you guys want to do to the format I got going so far?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,41 +1,87 @@
...
 				</section><br><br>
+                
+                <!-- VU //////////////////////////////////// -->
 				<section>	
-					section 2
+                    <h1>Mike Tyson</h1>
+                    
+                    <img src="images/mike.jpg" alt="mike tyson">
+                    
+			p>Michael Gerard Tyson was born on June 30, 1966 in Brooklyn, New York to a young Jimmy Kirkpatrick and Lorna Tyson.
```
Comment (WChihWen) [inline index.html]: There is a < p > tag, but it lost "<".

**Review by WChihWen — CHANGES_REQUESTED** [review]
The 42nd line lost "<"

**Review by catclops — COMMENTED** [review]
Agreed with Chih. I looked over the code and all I could see was the missing "<" for line 42.

**Review by Squirrel10 — APPROVED** [review]
I like the look of the page!

File: index.html (hunk 1 above)
Comment (krayzieeddi) [reply index.html]: ok i added in the last < to it.

**Review by WChihWen — APPROVED** [review]
It's good to update. I'll merge it.

**Pull request merged**